EFCore.CheckConstraints/Internal/DiscriminatorCheckConstraintConvention.cs

```diff
@@ -29,18 +30,23 @@ public virtual void ProcessModelFinalizing(IConventionModelBuilder modelBuilder,
                 .Where(g => g.Key.GetDiscriminatorProperty() != null && g.Key.GetIsDiscriminatorMappingComplete())
                 .Select(g => (g.Key, g.Select(e => e.GetDiscriminatorValue()))))
             {
+                if (!(rootEntityType.GetTableName() is string tableName))
+                {
+                    continue;
+                }
+
+                var tableIdentifier = StoreObjectIdentifier.Table(tableName, rootEntityType.GetSchema());
```

Comment:
By calling `StoreObjectIdentifier.Create(rootEntityType, StoreObjectType.Table)` you can combine this and the previous calls.